config/autoload/routes.global.php

```diff
@@ -147,6 +150,33 @@
             'allowed_methods' => ['GET', 'POST'],
         ],
         [
+            'name' => 'account-locations-list',
+            'path' => '/account/locations',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\ListLocationsAction::class,
+            ],
+            'allowed_methods' => ['GET'],
+        ],
+        [
+            'name' => 'account-location-add',
+            'path' => '/account/location/add',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\AddLocationAction::class,
+            ],
+            'allowed_methods' => ['GET', 'POST'],
+        ],
+        [
+            'name' => 'account-location-edit',
+            'path' => '/account/location/{uuid}/edit',
```

Comment:
It's probably a little excessive - yes it's do-able, but the regex for a UUID is quite long. Would it slow down routing much I wonder?